code/modules/client/preference_setup/loadout/items/xeno/unathi.dm

```diff
@@ -4,7 +4,7 @@
 	regions of Moghes. The forest mantle is exclusively for nobility these days."
 	path = /obj/item/clothing/accessory/poncho/unathimantle
 	cost = 1
-	whitelisted = list(SPECIES_UNATHI, SPECIES_VAURCA_WORKER, SPECIES_VAURCA_WARRIOR)
+	whitelisted = list(SPECIES_UNATHI, SPECIES_VAURCA_WORKER, SPECIES_VAURCA_WARRIOR, SPECIES_HUMAN, SPECIES_SKRELL, SPECIES_SKRELL_AXIORI)
```

Comment:
I do not believe there is a human whitelist. You may wish to remove this line entirely instead.

Reply:
whitelisted is the species that can take it, it has nothing to do with the actual whitelist.
items can be locked to humans, and many are.